src/main/resources/config.yml

```
@@ -55,14 +55,32 @@ DisabledSkins:
     - "Steve"
     - "Owner"
 
-# Custom list for the /skins GUI.
-# ShowOnlyCustomGUI will only show CustomGUI.Names in the gui.
-CustomGUI:
-  Enabled: false
-  ShowOnlyCustomGUI: true
-  Names:
-    - "xknat"
-    - "McLive"
+# GUI settings
+SKINSGUI:
```

```suggestion
SkinsGUI:
```
I think that's how it should be.